tests/Fixtures/Factories/PostFactory.php

```diff
@@ -2,13 +2,33 @@
 
 namespace Zenstruck\Foundry\Tests\Fixtures\Factories;
 
+use Zenstruck\Foundry\FactoryCollection;
 use Zenstruck\Foundry\Instantiator;
 use Zenstruck\Foundry\Persistence\PersistentObjectFactory;
+use Zenstruck\Foundry\Proxy;
+use Zenstruck\Foundry\RepositoryProxy;
 use Zenstruck\Foundry\Tests\Fixtures\Entity\Post;
+use Zenstruck\Foundry\Tests\Fixtures\Repository\PostRepository;
 
 /**
  * @author Kevin Bond <kevinbond@gmail.com>
  * @extends PersistentObjectFactory<Post>
+ *
+ * @method static Post|Proxy                     createOne(array $attributes = [])
+ * @method static Post[]|Proxy[]                 createMany(int $number, array|callable $attributes = [])
+ * @method static Post[]&Proxy[]                 createSequence(array|callable $sequence)
+ * @method static Post|Proxy                     find(object|array|mixed $criteria)
+ * @method static Post|Proxy                     findOrCreate(array $attributes)
+ * @method static Post|Proxy                     first(string $sortedField = 'id')
+ * @method static Post|Proxy                     last(string $sortedField = 'id')
+ * @method static Post|Proxy                     random(array $attributes = [])
+ * @method static Post|Proxy                     randomOrCreate(array $attributes = []))
+ * @method static Post[]|Proxy[]                 all()
+ * @method static Post[]|Proxy[]                 findBy(array $attributes)
+ * @method static Post[]|Proxy[]                 randomSet(int $number, array $attributes = []))
+ * @method static Post[]|Proxy[]                 randomRange(int $min, int $max, array $attributes = []))
```

Comment:
I added those, because some were creating problems with phpstan, now I'm also analyzing this factory with `make sca`